.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
It's quite long, but we have confidence that there is a small possibility someone uses this name somewhere else.